### ISSUE TITLE
Accessibility improvements - FITB activity

### ISSUE THREAD

**Issue by MillieMD** [MillieMD/revs-cocktail-quiz#6]
From WAVE: 
- Missing labels for all inputs

Other:
- Text too small on mobile
- Button too small on mobile
- Button has contrast issues on some parts of both gradients

**Comment by MillieMD** [comment]
linked branch

**Comment by MillieMD** [comment]
Looking at this code further, I think alot of the code for generating the activities needs rewriting to make better use of semantic HTML - some stuff can be done as quicker fixes (e.g. `<p>` -> `<label>`) but the whole structure of this could make use of fieldsets to better communicate the structure to screen readers

**Comment by MillieMD** [comment]
Another issues: red and green text in the input box does not meet WCAG requirements

**Comment by MillieMD** [comment]
> Another issues: red and green text in the input box does not meet WCAG requirements

Red contrast ratio: 4.44
Green contrast ratio: 1.66
